config/locales/en.yml

```diff
@@ -4,3 +4,5 @@ en:
     pose: '%{name} %{message}'
     semipose: '%{name}%{message}'
     emit: '<%{name}> %{message}'
+    location_name: 'You are in %{message}'
```

Comment:
The `emits` namespace is for messages emitted into a room. How about `location.name` and `location.description`?
Also, the "You are in" prefix is an interesting thought. I'm more used to games that just display the room name with a little ANSI color (usually highlighted bright white) without a prefix, so that's what I had in mind, but I can be persuaded otherwise.